GoogleForCreators/web-stories-wp
# Layer Panel: Add sticker layer styles

## Context

Stickers got added after the layer panel update, so they needed some updates as well.
[Designs](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3360%3A63235)

## Summary

- Updates sticker layer styling
- Updates layer panel styling to use grid

## Relevant Technical Choices

- the layer text in the panel wasn't lining up properly, so I switched from flexbox to grid. Could have been done in flexbox too
- stickers don't have a title, so I labelled them all as `Sticker`

## To-do

none

## User-facing changes

|Before|After|
|--|--|
|![before](https://user-images.githubusercontent.com/22185279/113943952-f2536a00-97c0-11eb-9b83-974ca33847b7.gif)|![after](https://user-images.githubusercontent.com/22185279/113943954-f2ec0080-97c0-11eb-86e8-413573c402b2.gif)|
|![ltr](https://user-images.githubusercontent.com/22185279/113944276-845b7280-97c1-11eb-8530-71189ed8e358.gif)|![before-ltr](https://user-images.githubusercontent.com/22185279/113944273-832a4580-97c1-11eb-981a-54011a28cd11.gif)|


## Testing Instructions

Stickers aren't user facing yet, but the best way to see them with minimal effort is:
1. Open dashboard
2. Go to explore templates
3. Pick the 'beauty' template (first one)
4. In editor, go to second page

Should see the stickers there.

### QA






This PR can be tested by following these steps:

Stickers aren't user facing yet, but the best way to see them with minimal effort is:
1. Open dashboard
2. Go to explore templates
3. Pick the 'beauty' template (first one)
4. In editor, go to second page

Should see the stickers there.

### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #7134
